tests/test_run_reference_vmm.py

```diff
@@ -67,14 +67,23 @@ def default_disk():
     )
 
 
+@pytest.fixture
```

Comment:
A general comment : While these changes are perhaps semantically more correct, I find them a bit kludgy, so if the general opinion is not to take these, we can drop these. A simple `setup_module` approach does not work. This is the least annoying way I could think of : -)

Reply:
I am not sure these are making the code more readable. I would drop them, and leave the code as it was before. @lauralt what do you think?

Reply:
So the "only" advantage with this approach is saving time when you first run the tests (and when you run them again there's no time difference between this solution and the way it was before)? If yes, I would not add these fixtures.

Otherwise, PR looks good to me. Thanks for the contribution!

Reply:
I'll drop the commit then!